src/components/spatial/Spatial.js

```diff
@@ -220,6 +258,28 @@ export default class Spatial extends AbstractSelectableComponent {
     });
   }
 
+  createRasterLayer() {
+    const source = this.raster;
+    const { colorValues, sliderValues, channelsOn } = this.state;
+    if (colorValues && sliderValues && channelsOn && source.channels) {
+      if (Object.keys(source.channels).length === Object.keys(colorValues).length
+        && Object.keys(source.channels).length === Object.keys(sliderValues).length
+        && Object.keys(source.channels).length === Object.keys(channelsOn).length) {
+        const props = {
+          useTiff: true,
+          useZarr: false,
+          sourceChannels: source.channels,
+          colorValues,
+          sliderValues,
+          channelsOn,
+
+        };
+        return new MicroscopyViewerLayer(props);
+      }
+    }
+    return null;
```

Comment:
Is this needed?

Reply:
I think there's an eslint rule about this.  I could ignore the rule - let me know what you prefer.